mopidy/core/playback.py

```diff
@@ -560,3 +572,26 @@ def _trigger_seeked(self, time_position):
         # TODO: Trigger this from audio events?
         logger.debug('Triggering seeked event')
         listener.CoreListener.send('seeked', time_position=time_position)
+
+    def _export_state(self):
+        """Internal method for :class:`mopidy.Core`."""
+        return models.PlaybackState(
+            tlid=self.get_current_tlid(),
+            position=self.get_time_position(),
+            state=self.get_state())
+
+    def _restore_state(self, state, coverage):
+        """Internal method for :class:`mopidy.Core`."""
+        if state:
+            new_state = None
+            if 'play-always' in coverage:
```

Comment:
`play-always` is not mentioned in the `Core._restore_state()` docstring.

It is also not part of the default coverage, so until the feature is made more configurable, this is dead code, AFAICS. Hopefully it is covered by tests.